apps/src/applab/DataProperties.jsx

```diff
@@ -3,47 +3,109 @@
  */
 
 import { DataView } from './constants';
+import Radium from 'radium';
 import React from 'react';
 import { changeView } from './redux/data';
-import color from '../color';
 import { connect } from 'react-redux';
-
-const styles = {
-  link: {
-    color: color.purple,
-    fontFamily: "'Gotham 7r', sans-serif"
-  }
-};
+import dataStyles from './dataStyles';
 
 const DataProperties = React.createClass({
   propTypes: {
     // from redux state
     view: React.PropTypes.oneOf(Object.keys(DataView)),
+    keyValueData: React.PropTypes.object.isRequired,
 
     // from redux dispatch
     onViewChange: React.PropTypes.func.isRequired
   },
 
+  /**
+   * Formats the key/value pairs on this.props.keyValueData and returns
+   * them as an array of arrays [[key1, val1], [key2, val2] ...].
+   * @returns {Array}
+   */
+  getKeyValueData() {
+    return Object.keys(this.props.keyValueData).map(key => {
+      return [key, this.props.keyValueData[key]];
+    });
```

Comment:
This comment made me realize that `getKeyValueData()` is useless. I've taken what you suggested here and inlined it in `render`.